style/global.css

```
@@ -0,0 +1,17 @@
+:root {
+	--gray900: #111827;
+ 	--gray800: #1F2937;
+    --gray700: #374151;
+    --gray600: #4B5563;
+    --gray500: #6B7280;
+    --gray400: #9CA3AF;
+    --gray200: #E5E7EB;
+    --gray100: #F3F4F6;
+    --gray50: #F9FAFB;
+
+    --blue: #3692FF;
+    --bluehover:#1967D6;
+    --blueactive: #1251AA;
```

❗️ 수정요청
css 변수에서 공백을 사용하지 못하니 단어를 구분하기 위해 특정 명명규칙을 이용하셔서 가독성을 높이시는 것을 추천드려요!
일반적으로 많이 사용되는 케밥케이스 사용을 추천드립니다.

```suggestion
    --blue-hover:#1967D6; 
    --blue-active: #1251AA;
```

[naming css variables](https://jwdallas.com/posts/namingcssvariables/)